apps/src/music/player/MusicPlayer.ts

```diff
@@ -539,7 +539,7 @@ export default class MusicPlayer {
       return;
     }
 
-    for (const instrument of library.folders.filter(
+    for (const instrument of library.instruments.filter(
```

Comment:
I think this will need to be both `[...library.instruments, ...library.kits]`

Reply:
Great catch.  Fixed.